File: use-cases/school-timetabling/src/main/java/org/acme/schooltimetabling/persistence/TimeslotRepository.java
package org.acme.schooltimetabling.persistence;

import jakarta.enterprise.context.ApplicationScoped;

import org.acme.schooltimetabling.domain.Timeslot;

import io.quarkus.hibernate.orm.panache.PanacheRepository;

@ApplicationScoped
public class TimeslotRepository implements PanacheRepository<Timeslot> {

}

File: technology/kotlin-quarkus/src/main/kotlin/org/acme/kotlin/schooltimetabling/persistence/TimeslotRepository.kt
package org.acme.kotlin.schooltimetabling.persistence

import io.quarkus.hibernate.orm.panache.kotlin.PanacheRepository
import org.acme.kotlin.schooltimetabling.domain.Timeslot
import jakarta.enterprise.context.ApplicationScoped

@ApplicationScoped
class TimeslotRepository : PanacheRepository<Timeslot>
